api/src/main/java/org/openmrs/notification/MessagePreparator.java

```diff
@@ -20,6 +20,6 @@
 	 * @param template
 	 * @return the prepared <code>Message</code>
 	 */
-	public Message prepare(Template template) throws MessageException;
+	//public Message prepare(Template template) throws MessageException;
```

Comment:
Don't just comment this out, delete the whole file. (Note that this is going to generate a cascade of different things to delete or fix, but that is the point.)